bazel/external/msgpack.BUILD

```diff
@@ -2,15 +2,11 @@ licenses(["notice"])  # Apache 2
 
 cc_library(
     name = "msgpack",
-    srcs = glob([
-        "src/*.c",
+    hdrs = glob([
         "include/**/*.h",
         "include/**/*.hpp",
     ]),
     defines = ["MSGPACK_NO_BOOST"],
-    includes = [
-        "include",
-    ],
-    strip_include_prefix = "include",
```

Comment:
@mmorel-35 im guessing `includes` is required - wondering if we should also have `strip_include_prefix` - if not im a bit confused cc @ohadvano

Reply:
yeah - i defo think we should - just confused - i guess the `strip_include_prefix` was never necessary

is `includes` necessary now?